styles.css

```diff
@@ -0,0 +1,32 @@
+.categorielistandroles {
```

Comment:
CSS classes need to be "as unique as possible", so you probably want to go for a prefix here, for example `.report_rolessitemap-categorylistandrole`

BTW: Typo here -> category instead of categorie

It might also be a good idea to reduce styling as much as possible and use bootstrap classes in the mustache template

Reply:
fixed